beekeeper-path-scheduler-apiary/src/main/java/com/expediagroup/beekeeper/scheduler/apiary/filter/LocationOnlyUpdateListenerEventFilter.java

```diff
@@ -25,25 +25,25 @@
 import com.expediagroup.beekeeper.core.model.LifecycleEventType;
 
 @Component
-public class MetadataOnlyListenerEventFilter implements ListenerEventFilter {
+public class LocationOnlyUpdateListenerEventFilter implements ListenerEventFilter {
```

Comment:
I think maybe lets stick to the old name? It matches a similar filter I think that's in Shunting Yard. What do you think?

Reply:
I also think the name is a lot clearer. I think in the context of this project, this name makes sense. Is it important for the names don't match?

Reply:
Let's stick with it